.gitignore

```
@@ -12,3 +12,6 @@ dz1/package-lock.json
 .motd_shown
 .bashrc
 .profile
+vk-edu-frontend/
+simple-chat/node_modules
+simple-chat/build
```

Вот так если исправить, в любой поддиректории будут игнорироваться `node_modules` и `build`.

```suggestion
node_modules
build
```

---

Я ведь не так написал.